docs/developers/eks-demo.md

```diff
@@ -32,17 +32,17 @@ This example will introduce how to use AWS-OTel-Collector to send application tr
 }
 ```
 5. Choose **Review policy**.
-6. On the Review policy page, enter `EKS-AWSOTel` for the Name and choose **Create policy**.
+6. On the Review policy page, enter `EKS-ADOT` for the Name and choose **Create policy**.
 
-#### Attach EKS-AWSOTel IAM Role to worker nodes
+#### Attach EKS-ADOT IAM Role to worker nodes
```

Comment:
These docs should point to setting up IRSA instead of attaching the policies to worker nodes. Could we create an issue to track that?

Reply:
Sure, will Create an issue to track this.